build.py

```diff
@@ -142,7 +142,7 @@ def do_make(target=None, log=False):
 token = None
 job = None
 boot_cmd = None
-use_git = False
+use_git = True
```

Comment:
If you want to make this the default, then make sure that when -e is used, we set use_git = False
But can't you just use the -g flag which flips this bit?